### DISCUSSION TITLE
ReflectionCaller use sun.reflect.Reflection but jdk11 has removed.

### DISCUSSION THREAD

**Discussion by peojectwang** [looly/hutool#2284] [Q&A]
ReflectionCaller中使用了sun.reflect.Reflection。jdk11版本中已经对sun.reflect.*API进行了移出。如果我的项目升级到jdk11我应该将hutool升级到哪个版本

**Comment by looly** [comment]
Hutool新版本已经移除这个类了吧。

你简单使用`CallerUtil.getCaller`即可。
